api/src/main/java/org/accula/api/config/SecurityConfig.java

```diff
@@ -20,8 +21,9 @@
      * @return custom filter chain
      */
     @Bean
+    @NotNull
     public SecurityWebFilterChain securityWebFilterChain(
-            ServerHttpSecurity http) {
+            @NotNull final ServerHttpSecurity http) {
```

Comment:
This method is called by Spring, and the parameters are injected by Spring as well. I don't really think the annotation does make much sense.

Reply:
fixed